src/website/views/custom-elements/alert.pug

```diff
@@ -1,8 +1,9 @@
 ---
 title: Alert
-scripts:
-  - js/ce/ux-chi-ce
 ---
+script(type='module' src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
+script(nomodule='' src='../../js/ce/ux-chi-ce/ux-chi-ce.js')
```

Comment:
Please use pug rootPath variable instead of ../../

Reply:
Currently rootPath is in use only for layout PUGs. I tried to add metalsmith-rootPath for build:test:fixtures task of Gulp (for test views). Also tried to use rootPath in custom-elements website views, but somehow the variable doesn't reach .PUGs. I'll get back to this later